src/Discovery/DiscoveryActionBar.tsx

```diff
@@ -129,6 +241,42 @@ const DiscoveryActionBar = (props: Props) => {
         && (
           <Space>
             <span className='discovery-export__selected-ct'>{props.selectedResources.length} selected</span>
+            {
+              props.config.features.exportToWorkspace.enableDownloadZip
+              && (
+                <Button
+                  onClick={() => {
+                    if (props.user.username) {
+                      handleDownloadZipClick(
+                        props.selectedResources,
+                        props.setDownloadInProgress,
+                        props.setDownloadStatusMessage,
+                        props.config.features.exportToWorkspace.manifestFieldName,
+                        props.config.features.exportToWorkspace.downloadZipMaxSizeBytes || 250000000,
+                      );
+                    } else {
+                      handleRedirectToLoginClick();
+                    }
+                  }}
+                  type='text'
+                  disabled={props.selectedResources.length === 0 || props.downloadInProgress === true}
+                  icon={<DownloadOutlined />}
+                >
+                  {(
+                    () => {
+                      if (props.user.username) {
+                        if (props.downloadInProgress === true) {
```

Comment:
minor: you can consider adding a pending state to this button in this case https://ant.design/components/button/#components-button-demo-loading